d2core/d2scene/map_engine_testing.go

```diff
@@ -10,13 +10,15 @@ import (
 
 	"github.com/OpenDiablo2/OpenDiablo2/d2common/d2interface"
 
+	"os"
```

Comment:
@Ziemas, try to format this source file using `goimports`. That should group related packages together. In this case the `os` package would be placed alongside the other packages of the standard Go library.

Run:
```
go get golang.org/x/tools/cmd/goimports
goimports -w d2core/d2scene/map_engine_testing.go
```